runtime/parachains/src/paras_inherent.rs

```diff
@@ -646,22 +622,6 @@ impl<T: Config> Pallet<T> {
 	}
 }
 
-/// Assures the `$condition` is `true`, and raises
-/// an error if `$action` is `true`.
-/// If `$action` is `false`, executes `$alt` if present.
-macro_rules! ensure2 {
-	($condition:expr, $err:expr, $action:ident $(, $alt:expr)? $(,)?) => {
-		let condition = $condition;
-		if !condition {
-			if $action {
-				ensure!(condition, $err);
-			} else {
-				$($alt)?
-			}
-		}
-	};
-}
-
```

Comment:
:+1: I feel a lot better seeing this removed, it added quite some mental overhead combined with the const generic